demonstrations/tutorial_vqe_bond_dissociation.py

```diff
@@ -0,0 +1,389 @@
+learn how quantum computers can be used to calculate equilibrium bond lengths, activation energy
+barriers, and reaction rates. As illustrative
+examples, we use tools implemented in PennyLane to study diatomic bond dissociation and reactions
+involving the exchange of hydrogen atoms.
+
+
+
+##############################################################################
+
+Potential Energy Surfaces 
+---------------------------------------------------------------------
+
+`Potential energy surfaces (PES) <https://en.wikipedia.org/wiki/Potential_energy_surface>`_
+describe the equilibrium energy of molecules for different positions of
+its atoms. The concept originates from the fact that nuclei are much heavier than
+electrons and thus move more slowly, allowing us to decouple their motion. We can then solve for
+the electronic wavefunction with the nuclei clamped to their respective positions. This leads
+to a separation of the nuclear and electronic parts of the Schrödinger equation, meaning we only
+need to solve the electronic equation:
+
+.. math:: H(R)|\Psi \rangle =  E|\Psi\rangle.
+
+From this perspective arises the concept of the electronic energy of a molecule, :math:`E(R)`,
+as a function of nuclear coordinates :math:`R`. The energy :math:`E(R)` is the expectation value
+of the molecular Hamiltonian, :math:`E(R)=\langle \Psi_0|H(R)|\Psi_0\rangle`, taken
+over the ground state :math:`|\Psi_0(R)`. The potential energy surface is
+precisely this function :math:`E(R)`, which connects energies to different geometries of the
+molecule. It gives us a visual tool to understand chemical reactions by associating
+stable molecules (reactants and products) with local minima, transition states with peaks,
+and by identifying the possible routes for a chemical reaction to occur.
+
+To build the potential energy surface, we compute the energy for fixed positions of the nuclei,
+and subsequently move the nuclei in incremental steps, computing the new energies at each point.
+The obtained set of energies corresponds to a grid of nuclear positions and the plot of
+:math:`E(R)` gives rise to the potential energy surface.
+
+
+.. figure:: /demonstrations/vqe_bond_dissociation/pes.png
+    :width: 75%
+    :align: center
+
+    Illustration of a potential energy surface for a diatomic molecule
+
+##########################################################
+
+Bond dissociation in a Hydrogen molecule 
+----------------------------------------
+
+We begin with the simplest of molecules: :math:`H_2`. 
+The formation or breaking of the :math:`H-H` bond is also the simplest
+of all reactions:
+
+.. math:: H_2 \rightarrow H + H.
+
+Using a minimal `basis set <https://en.wikipedia.org/wiki/STO-nG_basis_sets>`_,
+this molecular system can be described by two electrons in four
+spin-orbitals. When mapped to a qubit representation, we need a total of four qubits.
+The `Hartree-Fock (HF) state is represented as :math:`|1100\rangle`, where the two
```

Comment:
```suggestion
The *Hartree-Fock (HF) state* is represented as :math:`|1100\rangle`, where the two
```
There is a rendering issue in this line due to a missing ` around Hartree-Fock, was this originally a link? In any case I've italicized it here to fix it but change it however you like.